### ISSUE TITLE
Trainer Missing Localization

### ISSUE THREAD

**Issue by WenXin20** [Pokecube-Development/Pokecube-Issues-and-Wiki#276]
### Issue Description:
- Some trainers are spawning in without localization.

#### What happens:
- A trainer randomly spawned in that was named `entity.pokecube_adventures.trainer.nitwit`

#### What you expected to happen:
- Trainer should have a proper name.

#### Steps to reproduce:

1. Have trainers spawn in randomly
...
____
#### Affected Versions:
- `Pokecube AIO: v2.0.11`
- `Minecraft: v1.15.2`
- `Forge: v31.1.39`
#

**Comment by Thutmose** [comment]
this is caused by a sync time issue, if you relog, you will notice the trainer has the correct name, I will try to get these dealt with eventually, but are currently low priority

**Comment by Thutmose** [comment]
these should be fixed for now, in the mean time, while waiting on a sync, they will just be labelled as "'Trainer"
